requirements/prod.txt

```diff
@@ -22,6 +22,7 @@ dj-database-url==0.2.2
 django-cache-nuggets==0.1.1
 django-cronjobs==0.2.3
 django_csp==2.0.3
+django-environ==0.4.0
```

Comment:
Looks like Travis fails to install this so the tests fail: https://travis-ci.org/mozilla/olympia/jobs/85553066#L251